<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoctorForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(245,222,179)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Patient Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_patientName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
   <item>
    <property name="text">
     <string>Choose Patient</string>
    </property>
   </item>
  </widget>
  <widget class="QTableView" name="tableView_patient_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>631</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_record">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="medical_record">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Check medical record</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
